examples/tts/conf/fastpitch_align_old.yaml

```diff
@@ -0,0 +1,173 @@
+learn_alignment: true
```

Comment:
Do we need this file?

Reply:
I would say yes. Because, loading checkpoints need a config file and we cannot use the same config file that is inside the old Fastpitch.nemo file. We can replace the config file inside the old Fastpitch.nemo file with this file, then we would not need this file anymore. Or, the user has to manually make quite a lot of changes to existing fastpitch_align_v1.05.yaml file to use it to load the old checkpoint.

Reply:
If we were using the old checkpoint, will it raise the ValueError that you added in the init? Or are there other issues?

Reply:
Yes, if we were using the old checkpoint then it will raise the ValueError that I added. The ValueError message points towards this config file and shows how to use it.

Reply:
> Can we print this error message

We are printing this error message when the user tries to load the old checkpoint with the `AudioToCharWithPriorAndPitchDataset` class.

> try using a config that creates the dataset properly

Do you mean an existing config? Yes, there are existing configs that use the new TTSDataset but other parts of the config are not compatible with the old checkpoint. 
That is why I added this config `fastpitch_align_old.yaml` which uses TTSDataset (the new dataset) to load the old model. This config is the config used to load the old checkpoint with just the dataset parts replaced by the TTSDataset settings. We can use a different name for this config if you want.

Reply:
Let's not add this yaml. Let's change this https://github.com/NVIDIA/NeMo/pull/3928/files#diff-eaf0251e9d829a2b3aa53d9d90c6ca8181cc52d02be139215cc8baffc986e7acR73-R77 to an error message.

Since we are only supporting inference with the old models, the only time that we call into the old dataset is in `parser()` so let's change it so that when a user calls into `parser()` with the old config, we then load a correct config (let's define this as a DictConfig or dictionary somewhere in code of just the dataset class rather than a full yaml config file) with TTSDataset. We can remove the the ValueError being raised in the init with a warning message saying no training/finetuning support and only inference.